### ISSUE TITLE
Rose of Oblivion is stoppable during Aura making but book says no

### ISSUE THREAD

**Issue by VallenFrostweaver** [Ellpeck/NaturesAura#28]
Nature's Aura v:1.12.2-1.5.1
Forge: 14.23.5.2781

The book reads: "During this process, it can neither be stopped, nor harvested."
I break the flower and it stopped (though it wasn't harvested.)

**Comment by Ellpeck** [comment]
"Cannot be harvested" means that it won't drop.

**Comment by VallenFrostweaver** [comment]
Not the harvesting. The stopping

**Comment by Ellpeck** [comment]
If you smash your coffee machine with a hammer, it will stop working, despite the fact that it might not have an off switch.

**Comment by VallenFrostweaver** [comment]
But the box said it wouldn't. I suggest removing the "stopped" comment to avoid confusion.

**Comment by Ellpeck** [comment]
How would a thing that's literally destroyed *not* stop working? It's very clearly implied in my opinion.

**Comment by VallenFrostweaver** [comment]
Ok, let me rephrase. Why are you saying it cannot be stopped?

**Comment by Ellpeck** [comment]
Because it solves the implication that, maybe, with a redstone signal or a second eye, you'd have the ability to stop it from releasing its aura without destroying it.

But you don't.
